ddl/check.lua

```diff
@@ -746,6 +784,47 @@ local function check_sharding_key(space)
     return true
 end
 
+local function check_sharding_data(space)
```

Comment:
I propose to call sharding key descriptor, sharding function, descriptors of the bucket_id field and the bucket_id index 'sharding metatada' (not 'data'). At least to be consistent in the terminology. But it also looks more natural: the whole database schema is data about data stored in the database: the database metadata.

Reply:
Updated:

```lua
local function check_sharding_metadata(space)
...
end
```